Documentation/ceph-upgrade.md

```diff
@@ -520,18 +523,85 @@ located.
         value: "quay.io/k8scsi/csi-snapshotter:v1.2.2"
     - name: ROOK_CSI_ATTACHER_IMAGE
         value: "quay.io/k8scsi/csi-attacher:v1.2.0"
+    - name: ROOK_CSI_RESIZER_IMAGE
+        value: "quay.io/k8scsi/csi-resizer:v0.4.0"
 ```
 
-### Verifying updates
+#### Verifying updates
 You can use the below command to see the CSI images currently being used in the cluster. Once there
 is only a single version of each image and the version is the latest one you expect, the CSI pods
 are updated.
 
 ```console
 # kubectl --namespace rook-ceph get pod -o jsonpath='{range .items[*]}{range .spec.containers[*]}{.image}{"\n"}' -l 'app in (csi-rbdplugin,csi-rbdplugin-provisioner,csi-cephfsplugin,csi-cephfsplugin-provisioner)' | sort | uniq
-quay.io/cephcsi/cephcsi:v1.2.2
+quay.io/cephcsi/cephcsi:v2.0.1
 quay.io/k8scsi/csi-attacher:v1.2.0
 quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
 quay.io/k8scsi/csi-provisioner:v1.4.0
 quay.io/k8scsi/csi-snapshotter:v1.2.2
+quay.io/k8scsi/csi-resizer:v0.4.0
+```
+### Updating cephcsi from v2.0.0 to v2.0.1
+
+If you have a v2.0 cluster with the following environment (`env`) variables:
+
+```yaml
+  env:
+    - name: ROOK_CSI_CEPH_IMAGE
+        value: "quay.io/cephcsi/cephcsi:v2.0.0"
+    - name: ROOK_CSI_REGISTRAR_IMAGE
+        value: "quay.io/k8scsi/csi-node-driver-registrar:v1.2.0"
+    - name: ROOK_CSI_PROVISIONER_IMAGE
+        value: "quay.io/k8scsi/csi-provisioner:v1.4.0"
+    - name: ROOK_CSI_SNAPSHOTTER_IMAGE
+        value: "quay.io/k8scsi/csi-snapshotter:v1.2.2"
+    - name: ROOK_CSI_ATTACHER_IMAGE
+        value: "quay.io/k8scsi/csi-attacher:v2.1.0"
+    - name: ROOK_CSI_RESIZER_IMAGE
+        value: "quay.io/k8scsi/csi-resizer:v0.4.0"
 ```
+
+Nodes with mounted Rook volumes will [hang after reboot](https://rook.io/docs/rook/master/ceph-common-issues.html#node-hangs-after-reboot).
```

Comment:
Links in the documentation should just be relative, and the links will be updated by the CI when the docs are published. For example, this should be the link:
```
[hang after reboot](ceph-common-issues.md#node-hangs-after-reboot)
```

Reply:
Since this issue is fixed now, can we add a note in that help topic that it is fixed in CSI v2.0.1 and newer?